### PR TITLE
update kind fm 6

### PR DESCRIPTION
**Pull request by StefonSimmons**
### What does this PR do? What is the motivation?

the kind param is no longer a user-defined frontmatter. it is built-in to be one of home, page, section [etc.](https://gohugo.io/methods/page/kind/)

removes kind frontmatter from several pages.

previews: (no changes expected):

jenkins integration pulled from external source should display: 
https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-6/integrations/?q=jenkin

see previews for  modified files in  comment below (no changes expected)
https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-6/
https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-6/ja
https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-6/ko
https://docs-staging.datadoghq.com/stefon.simmons/update-kind-fm-6/fr

### Merge instructions

ensure only the actually `kind` frontmatter is removed from the page and not `kind` frontmatter as content on the page (e.g. `kind` appearing in a code block)
- [x] Please merge after reviewing

### CODE REVIEW

**Review by bgdeutsch — APPROVED** [review]
LGTM 👍

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (StefonSimmons) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, I noticed a couple of files where code blocks are being modified instead of front matter:

- `content/fr/database_monitoring/setup_postgres/rds.mdcontent/fr/database_monitoring/setup_postgres/rds.md`
- `content/fr/database_monitoring/setup_sql_server/azure.md`

Also, just wanted to confirm about some changes which aren't mentioned in the description. Are these intentional?
- `event.hostname` parameter being added to `content/en/security/threats/linux_expressions.md` and `content/en/security/threats/windows_expressions.md`
- Some docs are switching from `kind` to `custom_kind`. `content/ko/integrations/1password.md` as one example.
- The `cache_enabled` parameter has been set to `false` in `local/bin/py/build/configurations/pull_config_preview.yaml`.

**Comment by StefonSimmons** [comment]
> Thanks, I noticed a couple of files where code blocks are being modified instead of front matter:
> 
> * `content/fr/database_monitoring/setup_postgres/rds.mdcontent/fr/database_monitoring/setup_postgres/rds.md`
> * `content/fr/database_monitoring/setup_sql_server/azure.md`
> 
> Also, just wanted to confirm about some changes which aren't mentioned in the description. Are these intentional?
> 
> * `event.hostname` parameter being added to `content/en/security/threats/linux_expressions.md` and `content/en/security/threats/windows_expressions.md`
> * Some docs are switching from `kind` to `custom_kind`. `content/ko/integrations/1password.md` as one example.

Thanks @buraizu! I updated the files you mentioned. The `custom_kind` frontmatter is correct. that fm should be applied to integrations so that the integration tile is displayed on the `/integrations` page

**Review by buraizu — APPROVED** [review]

**Pull request merged**